composer.json

```diff
@@ -26,7 +26,7 @@
         "pimple/pimple": "3.*",
         "robmorgan/phinx": "*",
         "silex/silex": "2.*",
-        "swiftmailer/swiftmailer": "5.0",
+        "swiftmailer/swiftmailer": "^6.0",
```

Comment:
Looking at the diff suggests that this was edited manually, as unrelated dependencies (for example, `phpunit/phpunit`) have been updated.

Can we revert the changes to `composer.lock` and run

```
$ composer require swiftmailer/swiftmailer:^6.0.2 --update-with-dependencies
```

instead?

Reply:
I wouldn't worry about that, tbh.